examples/integrations-playground/README.md

```diff
@@ -0,0 +1,9 @@
+# Astro + Lit Example
+
+```
+npm init astro -- --template framework-lit
+```
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/withastro/astro/tree/latest/examples/framework-lit)
+
+This example showcases Astro working with [Lit](https://lit.dev/).
```

Comment:
These are all outdated. Should point to and describe `integrations-playground`

Reply:
Good catch! I don't think this will survive much longer (unless people want it to?), so I'll just remove for now.